spring-boot-starter/spring-boot-starter-discovery/pom.xml

```diff
@@ -75,5 +75,9 @@
       <groupId>org.apache.servicecomb</groupId>
       <artifactId>service-registry</artifactId>
     </dependency>
+    <dependency>
```

Comment:
why need this new dependency?

Reply:
The DiscoveryFilter interface is defined in this package

Reply:
no , you add new core dependence

Reply:
That's for Const.RESTFUL